app/views/groups/creategroup.html

```diff
@@ -114,7 +114,7 @@
 											</div>
 										</div>
 										<div class="col-sm-1 col-md-1">
-											<button ng-show="available" type="button" class="btn angled-btn btn-primary" data-ng-click="add()">
+											<button ng-show="available" type="button" class="btn angled-btn btn-primary" data-ng-click="add()" title="Add Client">
```

Comment:
Instead of adding a Title, we could have a tooltip using `uib-tooltip`. Also, I guess we should use label expressions since the label might be needed to be translated in different languages. so I guess you could just use this label `{{ 'label.button.add' | translate }}` Same goes for everywhere where you made changes :)